File: src/main/java/com/gmail/borlandlp/minigamesdtools/arena/gui/providers/examples/scoreboard/ScoreboardExampleCreator.java
package com.gmail.borlandlp.minigamesdtools.arena.gui.providers.examples.scoreboard;

import com.gmail.borlandlp.minigamesdtools.arena.gui.providers.GUIProvider;
import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider;
import com.gmail.borlandlp.minigamesdtools.creator.Creator;
import com.gmail.borlandlp.minigamesdtools.creator.CreatorInfo;

import java.util.ArrayList;
import java.util.List;

@CreatorInfo(creatorId = "scoreboard_example")
public class ScoreboardExampleCreator extends Creator {
    @Override
    public GUIProvider create(String id, AbstractDataProvider dataProvider) throws Exception {
        return new ScoreboardExample();
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/arena/gui/providers/examples/scoreboard/ScoreboardExampleCreator.kt
package com.gmail.borlandlp.minigamesdtools.arena.gui.providers.examples.scoreboard

import com.gmail.borlandlp.minigamesdtools.arena.gui.providers.GUIProvider
import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider
import com.gmail.borlandlp.minigamesdtools.creator.Creator
import com.gmail.borlandlp.minigamesdtools.creator.CreatorInfo

@CreatorInfo(creatorId = "scoreboard_example")
class ScoreboardExampleCreator : Creator() {
    @Throws(Exception::class)
    override fun create(id: String, dataProvider: AbstractDataProvider): GUIProvider {
        return ScoreboardExample()
    }
}